browser/about_flags.cc

```diff
@@ -361,44 +361,40 @@
 #endif
 
 #if BUILDFLAG(ENABLE_AI_CHAT)
-#define BRAVE_AI_CHAT                                          \
-  EXPAND_FEATURE_ENTRIES({                                     \
-      "brave-ai-chat",                                         \
-      "Brave AI Chat",                                         \
-      "Summarize articles and engage in conversation with AI", \
-      kOsWin | kOsMac | kOsLinux | kOsAndroid,                 \
-      FEATURE_VALUE_TYPE(ai_chat::features::kAIChat),          \
-  })
-#define BRAVE_AI_CHAT_HISTORY                                \
-  EXPAND_FEATURE_ENTRIES({                                   \
-      "brave-ai-chat-history",                               \
-      "Brave AI Chat History",                               \
-      "Enables AI Chat History persistence and management",  \
-      kOsWin | kOsMac | kOsLinux,                            \
-      FEATURE_VALUE_TYPE(ai_chat::features::kAIChatHistory), \
-  })
-#define BRAVE_AI_CHAT_CONTEXT_MENU_REWRITE_IN_PLACE                      \
-  EXPAND_FEATURE_ENTRIES({                                               \
-      "brave-ai-chat-context-menu-rewrite-in-place",                     \
-      "Brave AI Chat Rewrite In Place From Context Menu",                \
-      "Enables AI Chat rewrite in place feature from the context menu",  \
-      kOsDesktop,                                                        \
-      FEATURE_VALUE_TYPE(ai_chat::features::kContextMenuRewriteInPlace), \
-  })
-#define BRAVE_AI_CHAT_PAGE_CONTENT_REFINE                                   \
-  EXPAND_FEATURE_ENTRIES({                                                  \
-      "brave-ai-chat-page-content-refine",                                  \
-      "Brave AI Chat Page Content Refine",                                  \
-      "Enable local text embedding for long page content in order to find " \
-      "most relevant parts to the prompt within context limit.",            \
-      kOsDesktop | kOsAndroid,                                              \
-      FEATURE_VALUE_TYPE(ai_chat::features::kPageContentRefine),            \
-  })
+#define BRAVE_AI_CHAT_FEATURE_ENTRIES                                        \
```

Comment:
This is changed because BRAVE_AI_CHAT would cause some naming collision.

Reply:
seems like this code should also have a feature flag?

Reply:
Added https://github.com/brave/brave-core/pull/26082/commits/5ab1ba2cff13f2be4aef37dd3acd3589fcaa96ec